benchmark/stream_buffer.cpp

```diff
@@ -11,9 +11,9 @@
 static void
 StreamBuffer_Construct(benchmark::State& state)
 {
-    for ([[maybe_unused]] const auto& _ : state) {
-        auto __ = quicr::StreamBuffer<std::uint8_t>();
-        benchmark::DoNotOptimize(__);
+    for ([[maybe_unused]] const auto& iteration : state) {
+        auto _ = quicr::StreamBuffer<std::uint8_t>();
+        benchmark::DoNotOptimize(_);
         benchmark::ClobberMemory();
```

Comment:
Probably better to maintain the state iteration as `_` and instead give a name to the temporary (which is a pattern used in the other benchmark files). The fact that it has to be marked as maybe_unused is super annoying given `_` is a common "unused" motif in other languages.